pyc8 Audit for DocumentStoreExample (#218)
Macrometacorp/docs: docs/collections/documents/document-store-example.md

<TabItem value="py" label="Python">

```py
from c8 import C8Client

HOST = 'gdn.paas.macrometa.io'
EMAIL = 'nemo@nautilus.com'
PASSWORD = 'xxxxx'
GEOFABRIC = '_system'
COLLECTION_NAME = 'employees'
FIELDS = ['email', 'name']

print("--- Connecting to C8")

client = C8Client(protocol='https', host=HOST, port=443,
                          email=EMAIL, password=PASSWORD,
                          geofabric=GEOFABRIC)

> print("Adding Hash Index...")

Reviewer: ```suggestion
print("Adding hash index...")
```

## Get realtime updates

Example for real-time updates from a collection in fabric:
:::note

> The 'Stream' parameter should be enabled within the collection to get realtime updates

Reviewer: ```suggestion
Enable the 'Stream' parameter within the collection to get real-time updates.
```
